src/sentry/ingest/consumer/simple_event.py

```diff
@@ -29,26 +30,43 @@ def process_simple_event_message(
       `symbolicate_event` or `process_event`.
     """
 
-    raw_payload = raw_message.payload.value
-    metrics.distribution(
-        "ingest_consumer.payload_size",
-        len(raw_payload),
-        tags={"consumer": consumer_type},
-        unit="byte",
-    )
-    message: IngestMessage = msgpack.unpackb(raw_payload, use_list=False)
+    try:
+        raw_payload = raw_message.payload.value
+        metrics.distribution(
+            "ingest_consumer.payload_size",
+            len(raw_payload),
+            tags={"consumer": consumer_type},
+            unit="byte",
+        )
+        message: IngestMessage = msgpack.unpackb(raw_payload, use_list=False)
 
-    message_type = message["type"]
-    project_id = message["project_id"]
+        message_type = message["type"]
+        project_id = message["project_id"]
 
-    if message_type != "event":
-        raise ValueError(f"Unsupported message type: {message_type}")
+        if message_type != "event":
+            raise ValueError(f"Unsupported message type: {message_type}")
 
-    try:
-        with metrics.timer("ingest_consumer.fetch_project"):
-            project = Project.objects.get_from_cache(id=project_id)
-    except Project.DoesNotExist:
-        logger.exception("Project for ingested event does not exist: %s", project_id)
-        return
+        try:
+            try:
+                with metrics.timer("ingest_consumer.fetch_project"):
+                    project = Project.objects.get_from_cache(id=project_id)
+            except Project.DoesNotExist:
+                logger.exception("Project for ingested event does not exist: %s", project_id)
+                return
+
+            return process_event(message, project, reprocess_only_stuck_events)
+        except Exception as exc:
+            raise Retriable(exc)
```

Comment:
just to be safe and avoid dlqing anything potentially retriable, we don't dlq any exception from the `process_event` function